webapp/src/hooks/hooks.js

```diff
@@ -23,6 +23,20 @@ export default class Hooks {
             this.store.dispatch(openCreateModalWithoutPost(description, contextArgs.channel_id));
             return Promise.resolve({});
         }
+
```

Comment:
nit: and can be a separate PR, but perhaps trim the white space and lowercase the command at the top of the function?